docs/multiple-namespaces.md

```diff
@@ -0,0 +1,111 @@
+# Multiple Namespace Support
+
+#### Motivation
+Kubernetes [Namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+make it possible for a Kubernetes cluster to be partitioned and allocated to
+sub-groups of a larger team. These sub-teams can then deploy and manage
+infrastructure with finer controls of resources, security, configuration etc.
+Kubernetes allows for one or more ingress resources to be defined independently
+within each namespace.
+
+As of version 0.7 [Azure Application Gateway Kubernetes
+Ingress Controller](https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/README.md) (AGIC) can ingest events from and observe multiple namespaces.
+Should the AKS administrator decide to use [App Gateway](https://azure.microsoft.com/en-us/services/application-gateway/) as an ingress, all
+namespaces will use the same instance of App Gateway. A single installation of
+Ingress Controller will monitor accessible namespaces and will configure the App
+Gateway it is associated with.
+
+#### Defaults
+By default, as of version 0.7, AGIC is configured to observe ingress resources
+from **default** namespace only. This is configured in
```

Comment:
AGIC is configured to observe ingress resources from **default** namespace or namespace specified in the helm config.